File: solutions/CPP/LinkedList/utils/uimanager.cpp
#include"filemanager.cpp"

class UIManager
{
    public:

        void showMenu(){
            cout<<"***** Menu *****"<<endl ;
            cout <<"1. insert new node "<<endl ;
            cout <<"2.search node "<<endl ;
            cout <<"3.display all nodes  "<<endl ;
            cout <<"4. remove node "<<endl ;
            cout<<"5.save and exit "<<endl ;
        
        }
        int getChoice (int choice ){
            cout <<"enter your choice "<< endl ; 
            cin >> choice;
            return choice;
        }
};

File: solutions/CS/linkeList/utils/UIManager.cs
namespace linkeList.utils;
public class UIManager
{
    public void ShowMenu()
    {
        System.Console.WriteLine("1. Insert Node");
        System.Console.WriteLine("2. Delete Node");
        System.Console.WriteLine("3. Display List");
        System.Console.WriteLine("4. Exit");
    }


    public int GetUserChoice()
    {
        System.Console.Write("Enter your choice: ");
        int choice = int.Parse(System.Console.ReadLine());
        return choice;
    }   
    // This class can be expanded to manage user interface related tasks
}